website/project/model.py

```diff
@@ -1362,6 +1364,10 @@ def save(self, *args, **kwargs):
         else:
             suppress_log = False
 
+        self.root = self._root._id
```

Comment:
It seems like this is correct - changing this to ```self._root```  it breaks almost all existing tests with the error ```DatabaseError: Record must be loaded.```